site/src/pages/CreateWorkspacePage/useWorkspaceDuplication.test.tsx

```diff
@@ -45,28 +49,23 @@ describe(`${useWorkspaceDuplication.name}`, () => {
     expect(result.current.isDuplicationReady).toBe(false);
 
     for (let i = 0; i < 10; i++) {
-      rerender({ workspace: undefined });
+      void rerender({ workspace: undefined });
```

Comment:
this really isn't supposed to be an `await`?

Reply:
Typo I forgot to fix – good catch